pandas/core/arrays/numpy_.py

```diff
@@ -51,6 +51,13 @@ def numpy_dtype(self) -> np.dtype:
         """
         return self._dtype
 
+    @property
+    def na_value(self) -> object:
+        if issubclass(self.type, np.floating):
```

Comment:
isn't this *always* nan?

Reply:
The problem was that `np.float64("nan") is not np.nan`. However, I've fixed this, so this method is actuallt not necessary.